gen/calc.py

```diff
@@ -163,6 +163,11 @@ def validate_mesos_log_retention_mb(mesos_log_retention_mb):
     assert int(mesos_log_retention_mb) >= 1024, "Must retain at least 1024 MB of logs"
 
 
+def validate_mesos_container_log_sink(mesos_container_log_sink):
+    assert mesos_container_log_sink in ['journald', 'logrotate', 'journald+logrotate'], \
```

Comment:
nit: it is better not to use backslash to indicate a new line in asserts but rather use parentheses

```
assert a == b, (
    'Error') 
```